packages/serverless/src/domain/use-case/database.ts

```diff
@@ -44,16 +46,21 @@ export async function getDatabases(
   pagination?: Pagination
 ): Promise<Database[]> {
   try {
-    const databasesInfo = await DatabaseEngine.getInstance()
-      .client.db()
-      .admin()
-      .listDatabases();
+    const dbEngine = DatabaseEngine.getInstance();
+    const databasesInfo = await dbEngine.client.db().admin().listDatabases();
```

Comment:
should be `listDatabase`

Reply:
That's mongodb code and it's `listDatabases`